package.json

```diff
@@ -67,6 +68,6 @@
   },
   "peerDependencies": {
     "headers-polyfill": "^3.0.4",
-    "msw": ">=0.40.2"
+    "msw": ">=0.44.0"
```

Comment:
Marking this change as `feat` so it'd be a breaking change. Bumping the `msw` peer dependency range to a breaking range is a breaking change for this package as well.

Reply:
ok, reworded the commit to `feat: update to msw 0.44.0 and migrated to MockedRequest`

Reply:
Thanks!